app/policies/photo_policy.rb

```diff
@@ -0,0 +1,19 @@
+# app/policies/photo_policy.rb
+
+class PhotoPolicy
+  attr_reader :user, :photo
+
+  def initialize(user, photo)
+    @user = user
+    @photo = photo
+  end
+
+  # Our policy is that a photo should only be seen by the
+  # owner or followers of the owner, unless the owner is not 
+  # private in which case anyone can see it
+  def show?
+    user == photo.owner ||
+      !photo.owner.private? ||
+      photo.owner.followers.include?(user)
+  end
+end
```

Comment:
I think you need to include the create, destroy, edit and update methods. I am getting an error when I click on 'Add Post'. In the methods I checked that the user was the photo owner

Reply:
agreed